packages/wpcom.js/examples/server/index.js

```diff
@@ -2,41 +2,42 @@
  * Module dependencies.
  */
 
-var express = require( 'express' );
-var WPCOM = require( '../../' );
+const express = require( 'express' );
+const path = require( 'path' );
+const WPCOM = require( '../../' );
 
 /**
  * wpcon app data
  */
 
-var wpapp = require( '../../test/config' );
+const wpapp = require( '../../test/config' );
 
 /**
  * Create a WPCOM instance
  */
 
-var wpcom = WPCOM();
+const wpcom = WPCOM();
 
 // setup middleware
 
-var app = express();
-var pub = __dirname + '/public';
+const app = express();
+const pub = path.join( __dirname, '/public' );
 app.use( express.static( pub ) );
 
-app.set( 'views', __dirname + '/' );
+app.set( 'views', path.join( __dirname, '/' ) );
 app.set( 'view engine', 'jade' );
 
 app.get( '/', function ( req, res ) {
 	// set site id
-	var site = wpcom.site( wpapp.site );
+	const site = wpcom.site( wpapp.site );
 
 	// get site info
 	site.get( function ( err, info ) {
 		if ( err ) return console.log( err );
 
 		// get lastest posts
-		site.postsList( { number: 10 }, function ( err, posts ) {
-			if ( err ) return console.log( err );
+		site.postsList( { number: 10 }, function ( error, posts ) {
+			if ( error ) return console.log( error );
```

Comment:
This `err` scope problem could also be fixed by promisifying the code. Creating an example of the `wpcom` promise API 🙂